FWCore/ParameterSet/python/Config.py

```diff
@@ -749,11 +860,20 @@ def _replaceInSequences(self, label, new):
             sequenceable.replace(old,new)
         for sequenceable in self.endpaths.itervalues():
             sequenceable.replace(old,new)
+    def _replaceInTasks(self, label, new):
+        old = getattr(self,label)
+        for task in self.tasks.itervalues():
+            task.replace(old, new)
+    def _replaceInSchedule(self, label, new):
+        if self.schedule_() == None:
+            return
+        old = getattr(self,label)
+        for task in self.schedule_()._orderedTasks:
+            task.replace(old, new)
     def globalReplace(self,label,new):
-        """ Replace the item with label 'label' by object 'new' in the process and all sequences/paths"""
+        """ Replace the item with label 'label' by object 'new' in the process and all sequences/paths/tasks"""
         if not hasattr(self,label):
             raise LookupError("process has no item of label "+label)
-        self._replaceInSequences(label, new)
```

Comment:
Yes. Before it was calling _replaceInSequences twice, once directly and once in setattr.